### ISSUE TITLE
Simplify secrets

### ISSUE THREAD

**Issue by SaschaMann** [exercism/github-actions#65]
https://github.blog/changelog/2022-05-03-github-actions-simplify-using-secrets-with-reusable-workflows/

**Comment by ahtashamjaved** [comment]
nice info thanks

**Comment by hsadia538** [comment]
@SaschaMann  I would Like to work on this, can you assign me ?

**Comment by SaschaMann** [comment]
@ErikSchierboom might be able to help you out, I'm not really active in this project anymore

**Comment by hsadia538** [comment]
@ErikSchierboom  can you verify?

**Comment by ErikSchierboom** [comment]
That is correct. I think there's just one location where this can be applied: https://github.com/exercism/org-wide-files/blob/main/tooling-files/.github/workflows/deploy.yml

We'd need to rename some settings for it to work at https://github.com/exercism/snippet-extractor/blob/main/.github/workflows/deploy.yml

**Comment by ErikSchierboom** [comment]
Because they use different names:

```
AWS_ECR_ACCESS_KEY_ID: ${{secrets.AWS_LAMBDA_ACCESS_KEY_ID}}
```

**Comment by artfuldev** [comment]
Hi, is this still useful and yet to be picked up? I have some time I can contribute to this issue with.

**Comment by ErikSchierboom** [comment]
I'm not entirely sure we want this. I sort of like the explicitness of the current setup.
